src/SOS/SOS.UnitTests/SOS.cs

```diff
@@ -21,7 +21,7 @@ public static class SOSTestHelpers
 {
     public static IEnumerable<object[]> GetConfigurations(string key, string value)
     {
-        return TestRunConfiguration.Instance.Configurations.Where((c) => key == null || c.AllSettings.GetValueOrDefault(key) == value).Select(c => new[] { c });
+        return TestRunConfiguration.Instance.Configurations.Where((c) => key == null || c.AllSettings.GetValueOrDefault(key) == value).DefaultIfEmpty(TestConfiguration.Empty).Select(c => new[] { c });
```

Comment:
Did vstest complain about null config? Or about an empty enum?

Reply:
vstest complained about an empty enumeration. So I put some somewhat ugly logic to never create/pass an empty enumeration.